swagger-templates/src/main/java/com/okta/swagger/codegen/AbstractOktaJavaClientCodegen.java

```diff
@@ -169,12 +171,64 @@ protected void buildTopLevelResourceList(OpenAPI openAPI) {
         resources.stream()
                 .map(resourceName -> openAPI.getComponents().getSchemas().get(resourceName))
                 .forEach(model -> {
+                    if(model.getExtensions() == null) {
+                        model.setExtensions(new HashMap<>());
+                    }
                     model.getExtensions().put("top-level", true);
+
+                    List<String> oktaTags = (List<String>) model.getExtensions().get("x-okta-tags");
+                    if (oktaTags != null) {
```

Comment:
Without a bit more cleanup I'm guessing we'll want to do few more special cases for now